src/Language/PureScript/CodeGen/JS/CoreFnOptimizer.hs

```diff
@@ -0,0 +1,49 @@
+module  Language.PureScript.CodeGen.JS.CoreFnOptimizer (optimizeCoreFn) where
+
+import Protolude ( Maybe(..), (<$>), identity, map )
+
+import Data.List (lookup)
+import Language.PureScript.AST.Literals (Literal(..))
+import Language.PureScript.AST.SourcePos (nullSourceSpan)
+import Language.PureScript.CoreFn.Ann (Ann)
+import Language.PureScript.CoreFn.Expr ( Bind, Expr(Literal, ObjectUpdate, Accessor) )
+import Language.PureScript.Label (Label(..))
+import Language.PureScript.Types (pattern REmptyKinded, Type(..))
+import Language.PureScript.Constants.Prim qualified as C
+import Language.PureScript.CoreFn.Module (Module(..))
+import Language.PureScript.CoreFn.Traversals (everywhereOnValues)
+
+------
+-- CoreFn Optimization for JS Codegen
```

Comment:
Squash these empty-line comments down a bit, probably just make this a doc comment for the top level optimize function (or the module at your discretion)

Reply:
Added a proper comment. Please tell, if there is any issues with updated changes

Reply:
LGTM 👍